examples/demo-app/src/components/sharing/cloud-storage.js

```diff
@@ -113,6 +113,8 @@ const ExportCloudModal = ({
   onCloudLoginSuccess
 }) => {
   const metaUrl = get(info, ['metadata', 'url']);
+  const error = get(info, ['error']);
```

Comment:
`get` should be added to `demo-app`'s dependency

Reply:
it's already in demo-app package.json: "lodash.get": "^4.4.2",